taccsite_cms/templates/assets_site_delayed.html

```diff
@@ -12,7 +12,7 @@
 <!-- Site Assets (Delayed): Scripts. -->
 {# FAQ: Bootstrap can use jQuery, Popper, then its own script #}
 {# SEE: https://getbootstrap.com/docs/4.0/getting-started/introduction/#starter-template #}
-<script src="https://code.jquery.com/jquery-3.2.1.slim.min.js" integrity="sha384-KJ3o2DKtIkvYIK3UENzmM7KCkRr/rE9/Qpg6aAZGJwFDMVNA/GpGFF93hXpG5KkN" crossorigin="anonymous"></script>
+<script src="https://code.jquery.com/jquery-3.2.1.min.js" integrity="sha384-xBuQ/xzmlsLoJpyjoggmTEz8OWUFM0/RC5BsqQBDX2v5cMvDHcMakNTNrHIW2I5f" crossorigin="anonymous"></script>
```

Comment:
[The `slim` jQuery is inadequate for form plugin. I think the size difference is negligible.](https://github.com/TACC/Core-CMS/commit/6834a1800ff778293912b515626677f47a21a036)

<sup>This tiny fix took many hours to figure out. 😭</sup>